### ISSUE TITLE
TLC finds stuttering where there shouldn't be any

### ISSUE THREAD

**Issue by jonesmartins** [tlaplus/tlaplus#727]
As per my [message](https://groups.google.com/g/tlaplus/c/k6tw6sf-uDU/m/IauhJ1tJAgAJ) on the TLA+ Google Group, TLC found stuttering where there shouldn't be any.

This happened both on Toolbox 1.7.1 and Toolbox 1.8.0.

Link to the Spec and config files: https://github.com/Isaac-DeFrain/TLAplusFun/tree/main/PeerDiscovery

With Toolbox 1.7.1 I tested setting "Init" and "Next" individually, and then setting "Spec". Both found stuttering.

The TLC version I used to check on the terminal was 2.18.

**Comment by lemmy** [comment]
Please zip/tar the `PeerDiscovery` directory including the nested `PeerDiscovery.toolbox` and share it with me.

**Comment by jonesmartins** [comment]
Here it is.

[PeerDiscovery.zip](https://github.com/tlaplus/tlaplus/files/8648165/PeerDiscovery.zip)

**Comment by lemmy** [comment]
Your Toolbox model defines the constant `Nodes` as `1..4`, while the command line has `1..5`.

**Comment by jonesmartins** [comment]
Well, that was very ~~stupid~~ silly of me, wasn't it? And it's not the first time, either.
Thanks for your patience, Markus. On a Sunday, even. It means a lot! 

You mentioned the Toolbox stores its settings at `/home/<user>/.tlaplus`. Would you think it's necessary to delete them as well when using the `--purge` flag or `purge` command on Linux (or any other OS)?

Also, may I suggest we add organization templates to this project to avoid problems such as these (as in me wasting your time)?

**Comment by lemmy** [comment]
I don't think that apt/dpkg purge usually removes configuration in users' home directories. It might be fine on a single-user system but likely makes users angry on a multi-user system.

Do you want to draft templates?
